functions.py

```diff
@@ -628,28 +634,29 @@ def get_to_base_unit_and_type_prefix(from_uri, query_uri):
     overlaps_sparql = """\
 PREFIX geo: <http://www.opengis.net/ont/geosparql#>
 PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
+PREFIX dcterms: <http://purl.org/dc/terms/>
+# PREFIX ipo: <http://purl.org/dc/terms/isPartOf> 
 PREFIX geox: <http://linked.data.gov.au/def/geox#>
-PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
+# PREFIX qb4st: <http://www.w3.org/ns/qb4st/>
 PREFIX epsg: <http://www.opengis.net/def/crs/EPSG/0/>
 PREFIX dt: <http://linked.data.gov.au/def/datatype/>
 SELECT <SELECTS>
 WHERE {
     {
         { 
-           ?s1 rdf:subject <URI> ;
-           <LINKSET_FILTER>
-           rdf:predicate geox:transitiveSfOverlap;
-           rdf:object ?o  .
-        } UNION {
+#           ?s1 rdf:subject <URI> ;
```

Comment:
I don't think we can take this out until we've rebuilt the linksets and that is beyond the scope of the LDAPI refactors currently underway. AFAIK we use geox:transitiveSfOverlap still and can't depend on geo:sfOverlaps yet

Reply:
Lets verify.

Reply:
so in tests with excelerator use cases, we either have to:
a) change the linksets and/or triples in the cache so that this API endpoint works with your proposed query @dr-shorthair 
or
b) defer removing the union statement (i.e. keep it) so it works with the current linkset triples

b) is more expedient; but a) is probably what we should do in the longer term